app/src/main/java/us/kikin/apps/android/bgplayer/ui/show/ShowListFragment.kt

```diff
@@ -17,15 +18,17 @@ import us.kikin.apps.android.bgplayer.ui.videos.VideoItemClickListener
 class ShowListFragment : Fragment(), VideoItemClickListener, ShowItemClickListener {
 
     private var _binding: FragmentShowBinding? = null
-    private val binding get() = _binding!!
+    private val binding get() = requireNotNull(_binding) {
+        throw IllegalStateException("Cannot access binding")
+    }
```

Comment:
I would just use `requireNotNull(_binding)` without the lambda if I were you. The custom error doesn't add much value since it's pretty clear what's wrong if `_binding` is null